src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,184 @@
+   - New Relic integrations
+metaDescription: Install our Ansible Automation Controller dashboards and see your Ansible data in New Relic.
+---
+
+import infrastructureAnsibleControllerDashboard from 'images/infrastructure_screenshot-full_ansible-controller-dashboard.png'
+
+With our Red Hat Ansible Automation Platform dashboards, you can easily track and visualize your Ansible Automation Platform logs, keep an eye on your jobs and monitor instance memory.
+
+Built with our infrastructure agent and our nri-prometheus integration, the Ansible Automation Platform integration gives you a set of pre-built dashboards that let you view your most critical query data, all in one place.
+
+<img
+  title="Ansible Controller"
+  alt="A screenshot depicting the Ansible Controller dashboard"
+  src={infrastructureAnsibleControllerDashboard}
+/>
+
+<figcaption>
+After setting up the integration with New Relic, see your data in dashboards like these, right out of the box.
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data so you can keep track of your app's performance.
+
+You can install the infrastructure agent in two different ways:
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+
+## Integrate Red Hat Ansible Automation Platform with New Relic [#integrate]
+
+Once installed, the infrastructure agent can ingest data from your app and send it to New Relic, but you still need to integrate – or establish a line of communication – between the agent and your app. Once integrated, you can get started with an out of the box monitoring solution for your Ansible Automation Platform app. 
+
+<CollapserGroup>
+<Collapser
+    id="step1"
+    title="Step 1. Create a bearer token on Ansible Automation Platform"
+    >
+Once logged into Ansible Automation Platform, navigate to the **Users** section, select **admin** and then add a **token** to the user. Copy the token and save it in your desired path.
+
+Here is an example token: 
```

Comment:
@homelessbirds I'd change this introductory sentence since these are commands you'd run to set up a token. You could even break these three commands into a short procedure like this:

1. Create the token file: touch /home/ansible-automation-patform/bearer_token_file
2. Edit the token file: nano /home/ansible-automation-platform/bearer_token_file
3. Paste your token into the file.
4. Save and exit the nano editor.

 I just noticed that pkosanam made a similar suggestion to this below.

Reply:
Thanks!